livekit-rtc/livekit/rtc/room.py

```diff
@@ -747,9 +747,11 @@ def _on_room_event(self, event: proto_room.RoomEvent):
                 event.stream_header_received.participant_identity,
             )
         elif which == "stream_chunk_received":
-            asyncio.gather(self._handle_stream_chunk(event.stream_chunk_received.chunk))
+            asyncio.create_task(
```

Comment:
@theomonnom in general this became an issue when switching from the RingQueue to asyncio.Queue as `put` is async with it. Maybe we should avoid this and switch back to RingQueue after all?

Reply:
We should keep a reference to every task that has been created.
https://docs.python.org/3/library/asyncio-task.html#asyncio.create_task

Otherwise it may no be executed completely.
Let's do the same implementation as what Ben did for RPC (This should also be cancelled on room disconnect I guess)